modules/expectations/mobile_expectations.lua

```diff
@@ -0,0 +1,42 @@
+local expectations = require('expectations')
+local events = require('events')
+
```

Comment:
@VVeremjova look like this file is related to session. Please rename it to session_expectations.lua